torch/csrc/jit/ir.cpp

```diff
@@ -330,13 +304,9 @@ void Node::lint() const {
     for (auto input : inputs_) {
       // WARNING: O(n^2)
       JIT_ASSERT(std::find(ALL_OF(input->uses_), Use(const_cast<Node*>(this), i)) != input->uses_.end());
+      size_t stage_ = stage();
```

Comment:
I can see we are butting heads about whether or not lint should use methods or direct data member access :) My philosophy was, "lint is making intensional statements about the actual data in the classes, not the extensional behavior of methods, therefore, methods should not be used as much as possible, and if you restructure how the internal data representation works, you SHOULD refactor the lint pass." But I am not hardline about this. You don't have to do anything here, but I thought I'd mention this in case you were wondering. It is a little goofy to have a local variable named `stage_` that is not a member variable though ;)

Reply:
Oh, actually that is a typo. Ill remove it. An earlier version of Node didn't have a stage_ and derived stage from its outputs, but I removed it because it was confusing for Nodes not to have an explicit stage.